src/ansys/pytwin/evaluate/model.py

```diff
@@ -43,20 +46,20 @@ def _log_message(self, msg: str, level: PyTwinLogLevel = PyTwinLogLevel.PYTWIN_L
 
     def _raise_model_error(self, msg):
         """
-        Over load this method in base class to provide your own exception class so that it can be caught explicitly.
+        Overload this method in the child class for supplying your own exception class to catch errors explicitly.
         """
         raise ModelError(msg)
 
     def _raise_error(self, msg):
         """
-        Use this method in base class to raise an error with meaningful message.
+        Provides a base class method for raising an error with a meaningful message.
```

Comment:
This looks like a method signature--not a class?

Reply:
This is a method of the base class. Does it make sense?